<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QDialog" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Oboe Settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Update subscriptions every</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="subscriptionUpdateRate">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <item>
      <property name="text">
       <string>10 minutes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15 minutes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30 minutes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 hour</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 hours</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 hours</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8 hours</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24 hours</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Jump forward amount</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="jumpForwardsTime">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>60</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Jump backward amount</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="jumpBackwardsTime">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>60</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="automaticDownloadCheckBox">
     <property name="text">
      <string>Automatically download new episodes</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
